src/api/simplybook/simplybook-api.ts

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+
+import { simplybookCompanyName, simplybookCredentials } from 'src/utils/constants';
+
+type Booking = {
+  client: {
+    name: string;
+    email: string;
+  };
+  code: string;
+};
+
+type SimplybookBookingInfo = {
+  clientEmail: string;
+  bookingCode: string;
+  date: Date;
+};
+
+const DATE_FORMAT_LENGTH = 'YYYY-mm-dd'.length;
+const SIMPLYBOOK_API_BASE_URL = 'https://user-api-v2.simplybook.me/admin';
+
+const getAuthToken: () => Promise<string> = async () => {
+  const response = await axios({
+    method: 'post',
+    url: `${SIMPLYBOOK_API_BASE_URL}/auth`,
+    data: JSON.parse(simplybookCredentials),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  return response.data.token;
+};
+
+const getBookingsForDate: (date: Date) => Promise<Booking[]> = async (date: Date) => {
+  const token = await getAuthToken();
+
+  const simplybookFilterDateString = date.toISOString().substring(0, DATE_FORMAT_LENGTH);
+
+  const bookingsResponse = await axios({
+    method: 'get',
+    url: `${SIMPLYBOOK_API_BASE_URL}/bookings?filter[date]=${simplybookFilterDateString}&filter[status]=confirmed`,
+    headers: {
+      'Content-Type': 'application/json',
+      'X-Company-Login': simplybookCompanyName,
+      'X-Token': `${token}`,
+    },
+  });
+
+  return bookingsResponse.data.data;
```

Comment:
I think this will error if the request fails. I don't think axios throws an error so either a try catch or a check for an error would be good?